code/game/machinery/camera/camera.dm

```diff
@@ -52,17 +52,20 @@
 /obj/machinery/camera/Destroy()
 	toggle_cam(null, 0) //kick anyone viewing out
 	if(assembly)
-		qdel(assembly)
-		assembly = null
+		QDEL_NULL(assembly)
 	if(istype(bug))
 		bug.bugged_cameras -= src.c_tag
 		if(bug.current == src)
 			bug.current = null
 		bug = null
-	qdel(wires)
-	wires = null
+	if(wires)
+		QDEL_NULL(wires)

```

Comment:
Is this *ever* used without an `if` above it? Why not just roll that into the `QDEL_NULL` macro? Even in the cases where it's really not necessary (which, to be honest, is all of them, since `qdel` ignores nulls), there should be no real performance impact.